src/components/products/ProductResultsBody.js

```diff
@@ -0,0 +1,205 @@
+            <div className="row mb-3">
+                <div className="col-lg-3 d-none d-lg-block">
+                    <select
+                        name="order"
+                        className="form-select"
+                        aria-label="Default select example"
+                        defaultValue=""
+                        onChange={handleInputChange}
+                    >
+                        <option value="">Select order</option> {/* Assuming you want a placeholder option */}
+                        <option value="price-asc">Price (lowest first)</option>
+                        <option value="price-desc">Price (highest first)</option>
+                        <option value="square-meters-asc">Surface (lowest first)</option>
+                        <option value="square-meters-desc">Surface (highest first)</option>
+                        <option value="year-built-asc">Year (lowest first)</option>
+                        <option value="year-built-desc">Year (highest first)</option>
+                        <option value="bathrooms-asc">Number of bathrooms (lowest first)</option>
+                        <option value="bathrooms-desc">Number of bathrooms (highest first)</option>
+                        <option value="bedrooms-asc">Number of bedrooms (lowest first)</option>
+                        <option value="bedrooms-desc">Number of bedrooms (highest first)</option>
+                    </select>
+
+                </div>
+                <div className="col-lg-9 col-xl-5 offset-xl-4 d-flex flex-row">
+                    <div className="input-group">
+                        <input
+                            className="form-control"
+                            type="text"
+                            placeholder="Search properties..."
+                            aria-label="search input"
+                        />
+                        <button className="btn btn-outline-dark">
+                            <FontAwesomeIcon icon={["fas", "search"]} />
+                        </button>
+                    </div>
+                    <button
+                        className="btn btn-outline-dark ms-2 d-none d-lg-inline"
+                        onClick={changeViewType}
+                    >
+                        <FontAwesomeIcon
+                            icon={["fas", viewType.grid ? "th-list" : "th-large"]}
+                        />
+                    </button>
+                    <button className="btn btn-outline-dark ms-2 d-none d-lg-inline" onClick={toggleMapView}>
+                        <FontAwesomeIcon icon="fa-regular fa-map" />
+                    </button>
+                </div>
+            </div>
+            {showMapView ? (<div className={"g-3 mb-4 flex-shrink-0"}>
+                {
+                    <div className={"h-100 w-100"}>
+                        <LoadScript googleMapsApiKey="AIzaSyDkWh3Nr1WBBY3lgfbzjyyZeVLRUwJ6U0w">
```

Comment:
Are you sure it is a good idea to put the API key in plaintext in a public GitHub repo?